exercises/knapsack/knapsack_test.py

```diff
@@ -0,0 +1,76 @@
+                                         {"weight": 10, "value": 21}]), 21)
+
+    def test_cannot_be_greedy_by_value(self):
+        self.assertEqual(solve_knapsack(10, [{"weight": 2, "value": 20},
+                                             {"weight": 2, "value": 20},
+                                             {"weight": 2, "value": 20},
+                                             {"weight": 2, "value": 20},
+                                             {"weight": 10, "value": 50}]), 80)
+
+    def test_example_knapsack(self):
+        self.assertEqual(solve_knapsack(10, [{"weight": 5, "value": 10},
+                                             {"weight": 4, "value": 40},
+                                             {"weight": 6, "value": 30},
+                                             {"weight": 4, "value": 50}]), 90)
+
+    def test_eight_items(self):
+        self.assertEqual(solve_knapsack(104, [{"weight": 25, "value": 350},
+                                              {"weight": 35, "value": 400},
+                                              {"weight": 45, "value": 450},
+                                              {"weight": 5, "value": 20},
+                                              {"weight": 25, "value": 70},
+                                              {"weight": 3, "value": 8},
+                                              {"weight": 2, "value": 5},
+                                              {"weight": 2, "value": 5}]), 900)
+
+    def test_fifteen_items(self):
+        self.assertEqual(solve_knapsack(750,
+                                        [{"weight": 70, "value": 135},
+                                         {"weight": 73, "value": 139},
+                                         {"weight": 77, "value": 149},
+                                         {"weight": 80, "value": 150},
+                                         {"weight": 82, "value": 156},
+                                         {"weight": 87, "value": 163},
+                                         {"weight": 90, "value": 173},
+                                         {"weight": 94, "value": 184},
+                                         {"weight": 98, "value": 192},
+                                         {"weight": 106, "value":  201},
+                                         {"weight": 110, "value":  210},
+                                         {"weight": 113, "value":  214},
+                                         {"weight": 115, "value":  221},
+                                         {"weight": 118, "value":  229},
+                                         {"weight": 120, "value": 240}]), 1458)
+
+    # Utility functions
+    def setUp(self):
+        try:
+            self.assertRaisesRegex
+        except AttributeError:
+            self.assertRaisesRegex = self.assertRaisesRegexp
+
+    def assertRaisesWithMessage(self, exception):
+        return self.assertRaisesRegex(exception, r".+")
```

Comment:
You don't need to include the Utility functions unless they are used. In this case, they are not.